Iniciante/1131/README.md

```diff
@@ -0,0 +1,84 @@
+# Problema:
+
+Escreva um programa para ler o número de gols marcados pelo Inter e pelo Grêmio em um GRENAL. Logo após escrever a mensagem "Novo grenal (1-sim 2-nao)" e solicitar uma resposta. Se a resposta for 1, o algoritmo deve ser executado novamente solicitando o número de gols marcados pelos times em uma nova partida, caso contrário deve ser encerrado imprimindo:
+
+- Quantos GRENAIS fizeram parte da estatística.
+- O número de vitórias do Inter.
+- O número de vitórias do Grêmio.
+- O número de Empates.
+- Uma mensagem indicando qual o time que venceu o maior número de GRENAIS (ou "Nao houve vencedor", caso termine empatado).
+
+##### Problema completo: https://www.urionlinejudge.com.br/judge/pt/problems/view/1131
+
+# Resolução:
+
+O objetivo desse exercício é comparar uma série de resultados de jogos de futebol entre Inter e Grêmio e mostrar quem venceu mais vezes. Para isso vamos definir o vencedor de cada partida e comparar o número de vitórias de cada time.
+
+Vamos precisar de 7 variáveis do tipo `int` nesse exercício:
+```c
+    int gol_gre, gol_int, v_gre, v_int, e, jogos, x;
+```
+As primeiras variáveis `gol_gre` e `gol_int` representam a quantidade de gols que Grêmio e Inter fizeram em um mesmo jogo, respectivamente. `v_gre` e `v_int` representam o número de vitórias de cada time, `e` representa o número de empates, `jogos` o número total de jogos disputados e `x` é a variável que será usada para saber se haverá outro jogo.
+
+A primeira coisa a se fazer para começar o algoritmo é igualar `v_gre`, `v_int`, `e` e `jogos` a 0, já que o número de jogos antes do início do programa é 0:
+```c
+    v_gre = 0;
+    v_int = 0;
+    e = 0;
+    jogos = 0;
+```
+Sem essa etapa os dados dessas variáveis seriam valores aleatórios criados com a variável.
```

Comment:
Acho que só a explicação do número de jogos antes do início do programa ser 0 com "assim como os pontos de cada time" já está legal, e não precisaria desse comentário dos valores aleatórios. Acho que essa parte dos valores aleatórios é bem baixo nível (por se tratar de alocação e tal) e talvez um iniciante não entenda muito bem. 
Essa é só uma sugestão, não tá errado errado então tudo bem